modules/nf-core/cooltools/eigscis/main.nf

```diff
@@ -0,0 +1,46 @@
+process COOLTOOLS_EIGSCIS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::cooltools=0.5.4 bioconda::ucsc-bedgraphtobigwig=377"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/mulled-v2-c81d8d6b6acf4714ffaae1a274527a41958443f6:cc7ea58b8cefc76bed985dcfe261cb276ed9e0cf-0' :
+    'quay.io/biocontainers/mulled-v2-c81d8d6b6acf4714ffaae1a274527a41958443f6:cc7ea58b8cefc76bed985dcfe261cb276ed9e0cf-0' }"
+
+    input:
+    tuple val(meta), path(cool), path(fasta), path(chromsizes)
+    val resolution
+
+    output:
+    tuple val(meta), path("${prefix}*")             , emit: results
+    tuple val(meta), path('*.bw')                   , emit: bigwig, optional: true
+    path("versions.yml")                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    def suffix = resolution ? "::/resolutions/$resolution" : ''
+    """
+    cooltools genome binnify --all-names ${chromsizes} ${resolution} > genome_bins.txt
+    cooltools genome gc genome_bins.txt ${fasta} > genome_gc.txt
```

Comment:
Do these command have args? If so, could you use `args2` and `args3` to specify these. Also what's the reason these are not separate modules?

Reply:
I will remove those lines.